charts/rasa-x/templates/rasa-x-deployment.yaml

```
@@ -90,6 +90,13 @@ spec:
             secretKeyRef:
               name: {{ template "rasa-x.secret" . }}
               key: "rasaXToken"
+        - name: "RASA_X_USERNAME"
+          value: "{{ $.Values.rasax.initialUser.username }}"
```

should work like this, no?
```suggestion
          value: "{{ .Values.rasax.initialUser.username }}"
```

---

Hm, it worked with $ as well. But will change it